src/include/rear/ui.rb

```diff
@@ -68,6 +68,23 @@ def UsedModules
       deep_copy(modules)
     end
 
+    # returns a list of mountpoints that have to be included in BACKUP_PROG_INCLUDE
+    def GetMountpoints
+      # -D excludes pseudo filesystems
+      # -r raw output, no ascii-art
+      # -n no column header
+      cmd = "findmnt -D -t notmpfs,nodevtmpfs -o TARGET -n -r | sort"
```

Comment:
what about using long options instead of short one and comments? so it can look like
```
cmd = "findmnt --df -t notmpfs,nodevtmpfs --output TARGET --noheadings --raw | sort"
```
and in fact `findmnt --help` do not enlist -t option at all